### PR TITLE
DDS Router Configuration with Forwarding Routes

### PR DESCRIPTION
**Pull request by Tempate**

### CODE REVIEW

File: docs/resources/tutorials/cloud/router_conf_with_forwarding_routes/router_conf_with_forwarding_routes.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  - src: ROS_2_Domain_0
+    dst:
+      - ROS_2_Domain_1
+
+  - src: ROS_2_Domain_1
+
+topic-routes:
+
+  - name: /secret
```
Comment (rsanchez15) [inline docs/resources/tutorials/cloud/router_conf_with_forwarding_routes/router_conf_with_forwarding_routes.yaml:23]: I'd say that this should be `rt/secret`. In any case, test this example using the compiled DDS Router you have.

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/resources/tutorials/cloud/router_conf_with_forwarding_routes/router_conf_with_forwarding_routes.yaml (hunk 1 above)
Comment (Tempate) [reply docs/resources/tutorials/cloud/router_conf_with_forwarding_routes/router_conf_with_forwarding_routes.yaml:23]: You're right. I tested it and it only works when I set the topic-routes's name to rt/secret.

**Review by rsanchez15 — APPROVED** [review]

**Comment by rsanchez15** [comment]
Do not merge until DDS Router v2.1.0 (with Forwarding Routes feature) is available in Vulcanexus!

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
Rebase and add [this warning](https://docs.vulcanexus.org/en/latest/rst/tutorials/cloud/router_topic_qos/router_topic_qos.html) on top of the tutorial. Then we can merge it.

**Review by rsanchez15 — APPROVED** [review]

**Comment by rsanchez15** [comment]
@mergifyio backport humble

**Pull request merged**